selfservice/hook/web_hook.go

```diff
@@ -174,26 +174,26 @@ func (e *WebHook) ExecutePostRecoveryHook(_ http.ResponseWriter, req *http.Reque
 			RequestCookies: cookies(req),
 			Identity:       session.Identity,
 		})
-	}, trace.WithAttributes(attribute.String("hook", KeyWebHook)))
+	})
 }
 
 func (e *WebHook) ExecuteRegistrationPreHook(_ http.ResponseWriter, req *http.Request, flow *registration.Flow) error {
-	return otelx.WithSpan(req.Context(), "selfservice.hook.ExecuteRegistrationPreHook", func(ctx context.Context) error {
+	return otelx.WithSpan(req.Context(), "WebHook.ExecuteRegistrationPreHook", func(ctx context.Context) error {
```

Comment:
I think we should include the package name in the trace

```suggestion
	return otelx.WithSpan(req.Context(), "selfservice.hook.WebHook.ExecuteRegistrationPreHook", func(ctx context.Context) error {
```

Alternatively, I think we could use the proper go module naming for the traces:

```
github.com/ory/kratos/selfservice/hook:WebHook.ExecuteRegistrationPreHook
```

Reply:
added